File: Luyencode/DK05 - Kiểm tra số chính phương/DK05.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    long long n;
    cin >> n;
    if(n < 0)
        cout << "NO";
    else
    {
        long long temp = (long long)sqrt(n);
        if(temp * temp == n)
            cout << "YES";
        else
            cout << "NO";
    }
    return 0;
}

File: Luyencode/DK05 - Kiểm tra số chính phương/DK05.cs
using System;
namespace SCP
{
    public class Program
    {
        static void Main(string[]args)
        {
            long n = long.Parse(Console.ReadLine());
            if(n < 0)
                Console.Write("NO");
            else
            {
                long temp = (long)Math.Sqrt(n);
                if(temp * temp == n)
                    Console.Write("YES");
                else
                    Console.Write("NO");
            }
        }
    }
}